plugins/nordic-hid/fu-nordic-hid-firmware-b0.c

```diff
@@ -16,20 +14,10 @@
 #define UPDATE_IMAGE_MAGIC_NRF53  0x00003502
 
 struct _FuNordicHidFirmwareB0 {
-	FuIhexFirmwareClass parent_instance;
-	guint32 crc32;
+	FuNordicHidFirmwareCommonClass parent_instance;
```

Comment:
Wouldn't FuNordicHidFirmwareClass be more usual? e.g. drop the `common` everywhere.